src/components/navigation/Footer.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react";
 import "./Footer.scss";
 import { Route, Routes, Link } from "react-router-dom";
```

Comment:
Can we remove the imports that we don't use in this file?

Reply:
Removed